content/en/docs/setup/platform-setup/MicroK8s/index.md

```diff
@@ -11,6 +11,12 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
+{{< warning >}}
```

Comment:
Thanks for doing this. Would it be better to use a snippet for this and just have it generate the banner if the date is too old?

Reply:
At least put the text in a boilerplate file, instead of adding multiple copies of the same text. https://preliminary.istio.io/latest/about/contribute/shortcodes/#use-boilerplate-text